sonar-plugin-api/src/main/java/org/sonar/api/component/RubyComponentService.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.sonar.api.component;
 
-import org.sonar.api.server.ServerSide;
-
 import javax.annotation.CheckForNull;
+import org.sonar.api.server.ServerSide;
 
 /**
  * @since 3.6
+ * @deprecated since 5.6
```

Comment:
contrary to what I told you before, I think this class should not be deprecated. We encourage people to smoothly replace Ruby on Rails by JS but in the current state RoR is not deprecated yet.